x-pack/plugin/ccr/qa/multi-cluster-with-security/roles.yml

```diff
@@ -2,7 +2,7 @@ ccruser:
   cluster:
     - manage_ccr
   indices:
-    - names: [ 'index1' ]
+    - names: [ 'index1', 'index3' ]
```

Comment:
I think more meaningful names are in order here? When I look at this and look at `FollowIndexSecurityIT` it takes some mental energy to keep track of what and where `index[1234]` are.